core/lib/server/pub-op.ts

```diff
@@ -114,6 +115,11 @@ type SetOptions = {
 	 * ```
 	 */
 	deleteExistingValues?: boolean;
+
+	/**
+	 * If true, nullish values will not be set.
+	 */
+	ignoreNullish?: boolean;
```

Comment:
So this flag will turn a `.set(slug, null)` into a no-op? Why delegate that responsibility to `PubOp` instead of the caller?